<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CutForm</class>
 <widget class="QWidget" name="CutForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>668</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="2,1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QGraphicsView" name="src_g"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLineEdit" name="src_p"/>
       </item>
       <item>
        <widget class="QPushButton" name="src_b">
         <property name="text">
          <string>选择的图片</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cut_b">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>分割</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGraphicsView" name="out_g_1"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLineEdit" name="out_p_1"/>
         </item>
         <item>
          <widget class="QPushButton" name="out_b_1">
           <property name="text">
            <string>保存</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGraphicsView" name="out_g_2"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="out_p_2"/>
         </item>
         <item>
          <widget class="QPushButton" name="out_b_2">
           <property name="text">
            <string>保存</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
